default/AI/AIstate.py

```diff
@@ -268,6 +273,39 @@ def assess_planet_threat(self, pid, sightingAge=0):
         infra = 0  # doesn't really contribute to combat since damage to shields & 'defense' done first
         return {'overall': defense*(defense + shields + infra), 'attack': defense, 'health': (defense + shields + infra)}
 
+    def log_immobile_fleet(self, fleet, system, rating):
```

Comment:
I suppose I don't really expect to need that anymore, and it would be easy enough to recreate if needed; it can be taken out.